interface/wx/spinctrl.h

```diff
@@ -145,6 +145,13 @@ class wxSpinCtrl : public wxControl
     */
     int GetValue() const;
 
+    /**
+        Get the value f increment fr a spin control.
```

Comment:
Something ate half of the letters `o` in this comment.

Reply:
@vadz ,
My keyboard on Windows laptop has this "feature".

But I'm surprised the spell checker didn't catch it...